### PR TITLE
My listings, edit a listing, delete a listing, global user context

### PR DESCRIPTION
**Pull request by ElektrosStulpas**
Also refactored ListingForm to be separate so it could be used by both CreateListing and EditListing, but couldn't figure out a good approach to show current image when editing, so that might be buggy

### CODE REVIEW

File: src/Components/MyListings.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                <tbody>
+                    {listings.length > 0 && (
+                        listings.filter(listing => {
+                            if (listing.ownerUsername === userGlobalState.userName)
+                                return listing;
+                        }).map(listing =>
+                            <tr key={listing.id}>
+                                <td key={listing.id}>{listing.title}</td>
+                                <td key={listing.id}>{listing.city}</td>
+                                <td key={listing.id}>{listing.description}</td>
```
Comment (accountas) [inline src/Components/MyListings.js:64]: Maybe react-boostrap button for consistent style?

File: src/Components/MyListings.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                        <th>Description</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {listings.length > 0 && (
+                        listings.filter(listing => {
+                            if (listing.ownerUsername === userGlobalState.userName)
+                                return listing;
+                        }).map(listing =>
+                            <tr key={listing.id}>
```
Comment (accountas) [inline src/Components/MyListings.js:61]: It might be possible to reuse listing card here?
But would need to add overwritable button to it

**Review by accountas — COMMENTED** [review]
LGTM

**Review by tautvilis — APPROVED** [review]

**Comment by Kazhiunea** [comment]
I agree with this PR

File: src/Components/MyListings.js (hunk 2 above)
Comment (ElektrosStulpas) [reply src/Components/MyListings.js:61]: honestly don't think seeing MyListings in grid view would be more convenient

**Pull request merged**